modules/bitgo/test/v2/unit/wallet.ts

```diff
@@ -1622,6 +1622,29 @@ describe('V2 Wallet:', function () {
         sendMany.should.deepEqual('sendTxResponse');
       });
 
+      it('should send many and call create transfer api', async function () {
+        const signedTransaction = {
+          txRequestId: 'txRequestId',
+        };
+
+        const prebuildAndSignTransaction = sandbox.stub(custodialTssWallet, 'prebuildAndSignTransaction');
+        prebuildAndSignTransaction.resolves(signedTransaction);
+        prebuildAndSignTransaction.calledOnceWithExactly(sendManyInput);
+
+        const sendTxRequest = sandbox.stub(TssUtils.prototype, 'sendTxRequest');
+        sendTxRequest.resolves('sendTxResponse');
+        sendTxRequest.calledOnceWithExactly(signedTransaction.txRequestId);
+
+        const createTransferNock = nock(bgUrl)
+          .persist().log(console.log)
```

Comment:
please remove `console.log()` - we should not be logging during test except while debugging locally.

Reply:
Sorry I added it when I was debugging. it's now removed